server/command_create.go

```diff
@@ -28,8 +25,8 @@ var createFlagSet *flag.FlagSet
 
 func init() {
 	createFlagSet = flag.NewFlagSet("create", flag.ContinueOnError)
-	createFlagSet.String("size", "100users", "Size of the Mattermost installation e.g. '100users' or '1000users'")
-	createFlagSet.String("version", "", "Mattermost version or Docker image e.g. '5.12.4' or 'mattermost/mattermost-enterprise-edition:5.12.5-rc1'")
+	createFlagSet.String("size", "miniSingleton", "Size of the Mattermost installation e.g. '100users' or '1000users'")
```

Comment:
```suggestion
	createFlagSet.String("size", "miniSingleton", "Size of the Mattermost installation e.g. 'miniSingleton' or 'miniHA'")
```

Probably better to suggest mini sizes overall